h/models/annotation.py

```diff
@@ -38,6 +38,11 @@ class Annotation(Base):
         types.URLSafeUUID, server_default=sa.func.uuid_generate_v1mc(), primary_key=True
     )
 
+    #: Annotation PK: this was added after annotation.id and it's meant to replace it short term
+    #: For now we'll have a sequence and a nullable serial column.
+    pk_sequence = sa.Sequence("annotation_id_seq", cycle=False)
```

Comment:
We don't want the column to be auto-incrementing right now, we just create the sequence for now to start filling the column with the task.

When we are done with that we can make the `server_default` of this column the sequence value like any other auto incrementing column.